### PR TITLE
msi builder improvements

### PR DESCRIPTION
**Pull request by markuskramerIgitt**
Include folders for `KEEP_CONFIG`:

    cache/salt/minion/extmods
    cache/salt/minion/files

Bugfix MINION_ID (MINION_HOSTNAME)
Start by default

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,51 +2,69 @@
...
- `MASTER_HOSTNAME`      | `salt`                  | The master hostname
+ `MASTER`               | `salt`                  | The master (name or IP). Only a single master. May also be read from kept config.
  `MASTER_KEY`           |                         | The master public key. See below.
- `MINION_HOSTNAME`      | `%COMPUTERNAME%`        | The minion id
- `START_MINION_SERVICE` | `0` (_false_)           | Whether to start the salt-minion service after installation
- `KEEP_CONFIG`          | `1` (_true_)            | keep configuration on uninstall. Only from command line
- `MINION_CONFIGFILE`    | `C:\salt\conf\minion`   | The minion config file       __DO NOT CHANGE (yet)__
+ `ZMQ_filtering`        | `False`                 | Set to `True` if the master requires zmq_filtering
+ `MINION_ID`            | Hostname                | May also be read from kept config.
+ `START_MINION`         | _undefined_             | Set to `1` to start the salt-minion service after installation
```
Comment (twangboy) [inline README.md]: I think default is to start the minion (or it should be)

File: README.md [hunk 2]
```diff
@@ -57,7 +75,7 @@ The target client is where the installer is deployed.
 
 ## Build client requirements
 
-The build client is where the installer is created.
+The build client is where the msi installer is build.
```
Comment (twangboy) [inline README.md]: built

File: README.md [hunk 3]
```diff
@@ -87,36 +105,39 @@ until `git status` returns
...
+      warning CNDL1150
+        2 Warning(s)
+        0 Error(s)
 
-A 'help' target is available which prints out all the targets, customizable
-properties, and the current value of those properties:
+## How to program the msi builder toolkit
 
-> msbuild msbuild.proj /t:help
+The remainder is documentation how to program the msi build toolkit.
```
Comment (twangboy) [inline README.md]: The following documents how to program the msi build toolkit

File: wix.d/MinionMSI/Product.wxs [hunk 4]
```diff
@@ -141,7 +166,12 @@
     <!-- Inspiration 1: https://www.hass.de/content/wix-how-use-removefolderex-your-xml-scripts -->
     <!-- Inspiration 2: http://robmensching.com/blog/posts/2010/5/2/the-wix-toolsets-remember-property-pattern/   Simple Remember Pattern SRP-->
     <!-- Learn: Condition with KEEP_CONFIG is useless on Components and on Features -->
-    <!-- SRP pattern 1/2 -->
+    <!-- SRP pattern
+       We cannot recursivly delete the var folder because it contains the cache, we we regard as configuration and optionally keep.
```
Comment (twangboy) [inline wix.d/MinionMSI/Product.wxs]: we regard as configuration

File: README.md (hunk 1 above)
Comment (markuskramerIgitt) [reply README.md]: I agree, as in nsis

Comment (markuskramerIgitt) [reply README.md]: Done in https://github.com/markuskramerIgitt/salt-windows-msi/commit/dc481f979a21de2690665346fc555b727405440d

File: README.md (hunk 2 above)
Comment (markuskramerIgitt) [reply README.md]: https://github.com/markuskramerIgitt/salt-windows-msi/commit/2ef91b131d91a4eaf1f176729d3e269bc0441be2

File: wix.d/MinionMSI/Product.wxs (hunk 4 above)
Comment (markuskramerIgitt) [reply wix.d/MinionMSI/Product.wxs]: https://github.com/markuskramerIgitt/salt-windows-msi/commit/56a0ce5ab97cb585a229779fe21b35d424ef11a2

**Comment by markuskramerIgitt** [comment]
@thanks for your feedback. All changes done.

**Pull request merged**